### PR TITLE
support configurable branch name

### PR DESCRIPTION
**Pull request by josephholsten**

### CODE REVIEW

**Comment by aliaksandrb** [comment]
Helpfull patch, @josephholsten , thank you! :+1:

**Comment by kbrock** [comment]
this looks nice. :+1:

**Comment by ashfurrow** [comment]
Nice! Thank you!

File: lib/middleman-gh-pages/tasks/gh-pages.rake [hunk 1]
```diff
@@ -3,10 +3,13 @@ require 'fileutils'
 def remote_name
   ENV.fetch("REMOTE_NAME", "origin")
 end
+def branch_name
```
Comment (afomi) [inline lib/middleman-gh-pages/tasks/gh-pages.rake]: whitespace

**Comment by afomi** [comment]
:+1:

**Comment by ashfurrow** [comment]
I've worked off of @josephholsten 's work to also add a task to deploy without building first, if that's helpful to anyone: https://github.com/ashfurrow/middleman-gh-pages/commit/e0d1da4bd945703ab207884a812cee8872071bdf

**Comment by wafcio** [comment]
:+1:

**Comment by rmm5t** [comment]
Good stuff. :beers:

**Pull request merged**